### PR TITLE
updated RF10 to current definitions

### PR DESCRIPTION
**Pull request by d70-t**
The yaml file has been created without the notebook, thus the notebook
became outdated and has been removed.

### CODE REVIEW

**Review by marc-prange — COMMENTED** [review]
* Looks very good overall. 
* I think it doesn't make so much sense to review the exact times right now, since the new dropsondes handling still needs to be implemented. I would prefer to do that once that has been done. 
With the traffic diversions in circle 4 and circle 6 I also don't see a better solution than writing an irregularity here, because splitting the circle up doesn't seem to be a good option. With the new dropsonde times, the original first dropsonde of circle 4 is excluded I think, watch out for that!
*Segments other than circles look good to me.

**Comment by d70-t** [comment]
@Smpljack RF10 should now comply with our new current definitions. Please have a look and consider merging.

File: flight_phase_files/HALO_RF10_20200207_info.yaml [hunk 1]
```diff
@@ -22,59 +23,330 @@ remarks:
...
+    - HALO-0207_s58
+    - HALO-0207_s59
+    - HALO-0207_s61
+    - HALO-0207_s62
+    BAD: []
+    UGLY: []
+- kinds: [circle_break]
+  name: circle break 5/6
+  start: 2020-02-07 19:26:59
+  end: 2020-02-07 19:39:43
```
Comment (marc-prange) [inline flight_phase_files/HALO_RF10_20200207_info.yaml]: Ends 2 seconds earlier than start of circle 6.

Comment (d70-t) [reply flight_phase_files/HALO_RF10_20200207_info.yaml]: changed it

**Review by marc-prange — COMMENTED** [review]
Just the one minor adjustment, looks great otherwise!

**Pull request merged**